### DISCUSSION TITLE
Will there be a Raspberry Pi Pico based ExpressLRS ?

### DISCUSSION THREAD

**Discussion by jaytxrx** [ExpressLRS/ExpressLRS#2001] [General]
I see that most of the diy ELRS TX circuits are using ESP32 boards. 
Will there be a TX board using Raspberry Pi Pico?

**Comment by Ncerzzk** [comment]
In my opinion, raspberry pico is not better than ESP32. Are there more advantanges?

**Reply by jaytxrx** [reply]
I just asked out of curiosity and I have some Pico lying around.
Are there any reasons why all TX boards are based on ESP32 ?

**Reply by pkendall64** [reply]
The Pico W would be a contender, just that no-one has done it. Also the libraries that we use would need a Pico port as well.
